api/routes/Teams.js

```diff
@@ -1,88 +1,19 @@
 const express = require('express')
 const router = express.Router();
 
-const team = require('../models/team');
-
-
-router.get('/',(req,resp,next) => {
-
-	team.findAll()
-		.then(teams => {
-			if(teams.length > 0){
-				console.log(teams)
-				resp.status(200).json(teams);
-			}
-			else {
-				resp.status(404).json({
-					message: "Entry Not Found"
-				})
-			}
-			
-		})
-		.catch(err => {
+const Team = require('../models/Team');
+
+router.get('/', async (req, res, next) => {
+	try {
+		teams = await Team.findAll();
+		if (teams.length > 0) {
+			return res.json(teams);
+		}
+		res.status(404).json({message: "No teams found"});
+	} catch(err) {
 			console.log(err);
-			resp.status(500).json({
-				error:err 
-		});
-
-	})
-});
-
-
-
-
-
-router.post('/',(req,resp,next) => {
```

Comment:
I took the remaining endpoints out, since we eventually want to move them to an /admin space.

Reply:
awesome